salt/apache/init.sls

```diff
@@ -57,3 +57,15 @@ apache2-utils:
 {{ apache(name, entry) }}
 {% endfor %}
 {% endif %}
+
+# Delay the Apache start up if the server has multiple IP addresses.
+{% if salt['pillar.get']('apache:wait_for_networking') == True %}
+/etc/systemd/system/apache2.service.d/customization.conf:
+  file.managed:
+    - source: salt://core/systemd/files/apache2.conf
+    - makedirs: True
+  module.run:
+    - name: service.systemctl_reload
+    - onchanges:
+      - file: /etc/systemd/system/apache2.service.d/customization.conf
```

Comment:
Would this work?

```suggestion
    - watch_in:
      - module: apache2-reload
```

Reply:
`apache2-reload` doesn't work for this change. 
We need to reload the systemctl daemon to pick up on the new systemd file - we don't actually need to reload Apache.

Reply:
Looks good, then!